citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -334,6 +353,149 @@ <h4 class="panel-title">
+		let segments = href.split('/');
+		let movedToCollectionKey = segments[6];
+		let itemsKeys = [];
+		ui.helper.find('ul').children().each(function(){
+			itemsKeys.push($(this).text());
+			});
+		let itemsMoveData ={};
+		itemsMoveData["itemIds"] = itemsKeys;
+		itemsMoveData["collectionId"] = movedToCollectionKey;
+		$.ajax({
+			type: "POST",
+			url: 'items/move?[(${_csrf.parameterName})]=[(${_csrf.token})]',
+			data: JSON.stringify(itemsMoveData),
+			contentType : "application/json",
+			dataType: "json",
+			success: function (moveItemsAsyncResponse){
+		       pollMovedItemsStatuses(moveItemsAsyncResponse, collectionName);
+			 },
+			 error: function (moveItemsAsyncResponse) {
+			   showNotification(" Error while moving citations!", "danger", 10000);
+			 }
+		});
+		$('.icon-arrow-left').show();
+		let checkedBoxes = $('input[type="checkbox"]:checked');
+		checkedBoxes.each(function(){
+			$(this).prop('checked', false);
+			$(this).parents('tr').removeClass('selected');
+			});
+		}
+});
+});	
+
+function pollMovedItemsStatuses(response,collectionName){
+$.ajax({
+	type: 'GET',
+	url: 'items/move/'+response.taskID+'/status?[(${_csrf.parameterName})]=[(${_csrf.token})]',
+	contentType : "application/json",
+	dataType: "json",
+	success: function (moveItemsStatusResponse) {
+		if(moveItemsStatusResponse.taskStatus === "COMPLETE"){
+		if(moveItemsStatusResponse.response.successItems.length != 0){
+	 	let message = " "+moveItemsStatusResponse.response.successItems.length+ " Citations moved successfully to ' "+collectionName+" ' ! <br /> <br />"+ moveItemsStatusResponse.response.successItems.join("<br>");
+	 	showNotification(message, "success", 20000);
+		}
+		if(moveItemsStatusResponse.response.failedItems.length != 0){
+	 	let message =  " "+ moveItemsStatusResponse.response.failedItems.length+ " Citations not moved to ' "+collectionName+" ' ! <br /> <br />"+ moveItemsStatusResponse.response.failedItems.join("<br>") 
+	 						+ "<br /> <br /> Reasons: <br /> <br />" + moveItemsStatusResponse.response.failedMessages.map(x => x.code + " : " + x.message).join("<br>");
+	 				
+	 	showNotification(message, "danger", 20000);
+		}
+		if(moveItemsStatusResponse.response.unchangedItems.length != 0){
+		let message =  " "+ moveItemsStatusResponse.response.unchangedItems.length+ " Citations already present in ' "+collectionName+" ' ! <br /> <br />"+ moveItemsStatusResponse.response.unchangedItems.join("<br>");
+		showNotification(message, "info", 20000);
+		}
+		clearAsyncTask(moveItemsStatusResponse.taskID);
+		}else{
+			setTimeout(pollMovedItemsStatuses(moveItemsStatusResponse,collectionName),1000);
+		}
+		},
+	error: function (moveItemsStatusResponse) {
+	    	showNotification("Error while polling citations statuses!", "warning", 1000)
+			},
+});
+}
+
+function clearAsyncTask(taskID){
+	$.ajax({
+		type: 'GET',
+		url: 'items/move/task/'+taskID+'/clear?[(${_csrf.parameterName})]=[(${_csrf.token})]',
```

Comment:
get requests don't need csrf tokens; but this should not be a get request since it changes the server state.